stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        rate = wins / num_played if num_played &gt; 0 else NO_INFO_PRIOR
</pre><p> </p><p>I don't really suggest you change anything (maybe could mention it as a possible extension at most), but you could also try to blend the no info prior and the win rate here (rather than only one or the other), which would be a step in the direction of having some kind of Bayesian model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>